src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -3634,6 +3649,49 @@ bool ApplicationManagerImpl::IsSOStructValid(
   return true;
 }
 
+// Convert mobile_apis::SamplingRate::eType enum into
+// media_manager::SamplingRate enum
+static media_manager::SamplingRate ConvertSamplingRate(
```

Comment:
Please refer to my comment https://github.com/smartdevicelink/sdl_core/pull/2184#issuecomment-644639701. If I moved the conversion code in MediaManager, it would make the component dependent on mobile API. I preferred to avoid it.

Reply:
I updated MediaManager to use Mobile API enums, so these conversion functions are no longer necessary. Please kindly confirm my commit 8211590.